integration_tests/test_local.py

```diff
@@ -260,3 +263,22 @@ def test_blueprint_example(self, *_):
 
         # execute uninstall workflow
         self.uninstall_blueprint()
+
+    def test_hello_world_example(self, *_):
+        self.test_name = 'test_hello_world_example'
+        self.blueprint_path = \
+            './examples/cloudify-hello-world-example/openstack.yaml'
+        self.inputs = dict(self.client_config)
+        self.inputs.update(
+            {
+                'external_network_id': 'dda079ce-12cf-4309-879a-8e67aec94de4',
+                'name_prefix': 'hello_world',
+                'image': 'e41430f7-9131-495b-927f-e7dc4b8994c8',
+                'flavor': '2',
+            }
+        )
+        self.initialize_local_blueprint()
+        self.install_blueprint()
+        sleep(10)
+        # execute uninstall workflow
```

Comment:
Can we check the URL for a 200 return code?

Reply:
I will check it and add that